vercel/examples
# Update web3/sessions to become web3/auth

### Description

The sessions example was not super clear on what it helped people achieve. This is a more to the point example on a pain point experiences by multiple web3 projects we got in touch with.

Note: There is a usecase for a session without login with ethereum. To keep an healthy selection of web3 content I'll keep both after talking with Steven.

### Best Way to Test

1. Navigate to the [example](https://web3-authentication.vercel.app)
2. If you don't have metamask you will see a message asking you to install it
3. if you have metamask you will be able to login and logout
4. see that the address name updates

### Type of Change

- [x] example improvement


### New Example Checklist
